modules/KalturaSupport/components/dualScreen/imagePlayer.js

```diff
@@ -104,6 +106,28 @@
 		addBinding: function(){
 			var _this = this;
 
+			this.bind('KalturaSupport_ThumbCuePointsReady' , function(){
+				if (_this.getPlayer().isLive()){
+					//Forcing 1st slide is only on VOD entry
+					return;
+				}
+				// on non-autoplayed player - check if required to force-show the 1st CP
+				var chaptersRawData = _this.getCuePoints();
+				//Since imagePlayer doesn't have a uiconf element - copy the value from chapters plugin which has representation in the uiconf;
```

Comment:
@eitanavgil what do you mean it doesn't have representation? Anything under dualScreen is configured in the dualScreen plugin and passed to its child components when it initialize it